README.md

```diff
@@ -147,61 +164,35 @@ containerize.  It is recommended that most users use a version tag (e.g.
 |`cisagov/example:latest`| The most recent release image pushed to a container registry.  Pulling an image using the `:latest` tag [should be avoided.](https://vsupalov.com/docker-latest-tag/) |
 
 See the [tags tab](https://hub.docker.com/r/cisagov/example/tags) on Docker
-Hub for a list of all the supported tags.
+Hub for a list of all the supported tags. -->
 
 ## Volumes ##
 
 | Mount point | Purpose        |
 |-------------|----------------|
-| `/var/log`  |  Log storage   |
-
-## Ports ##
-
-The following ports are exposed by this container:
-
-| Port | Purpose        |
-|------|----------------|
-| 8080 | Example only; nothing is actually listening on the port |
-
-The sample [Docker composition](docker-compose.yml) publishes the
-exposed port at 8080.
+| `/home/cisa`  |  Log storage   |
 
 ## Environment variables ##
 
 ### Required ###
 
-There are no required environment variables.
-
-<!--
 | Name  | Purpose | Default |
 |-------|---------|---------|
-| `REQUIRED_VARIABLE` | Describe its purpose. | `null` |
--->
+| `PCA_GENERATOR_IMAGE` | Docker image name. | `cisagov/pca-report-generator` |
```

Comment:
`PCA_GENERATOR_IMAGE` is not an environment variable specified in the Dockerfile.  There are 3 `ENV` variables currently in there and they are all optional since they are all given default values.  Make sure that those variables are documented in the "Optional" section below.

Reply:
Resolved in ea18e214736be0c31a4e7e018469040e6686c693

Reply:
The environment variables should be listed here alphabetically.

Also, a reminder that the person who starts a conversation should be the one to mark it as "resolved".  The only exception to this is when you commit a suggestion- in that case, GitHub automatically marks the conversation as resolved by the committer, which is fine.

Reply:
Oh okay, np. Will make note of that.

Reply:
Resolved in 4d038c2f7b9eab871cc0906b28694563547d5a7c